src/solvers/dgmulti/flux_differencing.jl

```diff
@@ -598,11 +600,48 @@ function rhs!(du, u, t, mesh, equations, initial_condition, boundary_conditions:
 
   @trixi_timeit timer() "Jacobian" invert_jacobian!(du, mesh, equations, dg, cache)
 
-  @trixi_timeit timer() "sources terms" calc_sources!(du, u, t, source_terms,
+  @trixi_timeit timer() "source terms" calc_sources!(du, u, t, source_terms,
                                                      mesh, equations, dg, cache)
 
   return nothing
 end
 
+# Specializes on SBP (e.g., nodal/collocation) DG methods with a flux differencing volume
```

Comment:
Why is this required?

Reply:
Because `DGMultiFluxDiffSBP  <: DGMultiFluxDiff`.

Previously, entropy projection methods had a `rhs!` which specialized on `DGMultiFluxDiff{Union{<:Polynomial, <:GaussSBP}}`, while `SBP` discretizations simply used a different volume kernel in the `rhs!` for a traditional `DGMulti` discretization.